app/util.js

```diff
@@ -37,7 +37,7 @@ var util = {
     },
 
     getGlairInsightsLink: function (urlString, id, reportType) {
-        return urlString.substring(0, urlString.indexOf('visual')) + "administration/report-management/report/" + id + "/" + reportType;
+        return urlString.substring(0, urlString.indexOf('visual')) + "administration/report-management//report/" + id + "/" + reportType;
```

Comment:
still this url is hard coded....place it where all constant are defined...is that file created?